documentation/sdk/docs/how_tos/advanced_transactions/send_micro_transaction.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Send a Micro Transaction
+description: 'How to send a microtransaction'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- micro-transaction
+- send micro-transaction
+- microtransaction
+- send microtransaction
+- nodejs
+- python
+- rust
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import RustCode from '!!raw-loader!../../../../../sdk/examples/how_tos/advanced_transactions/send_micro_transaction.rs';
+import NodejsCode from '!!raw-loader!../../../../../bindings/nodejs/examples/how_tos/advanced_transactions/send_micro_transaction.ts';
+import PythonCode from '!!raw-loader!../../../../../bindings/python/examples/how_tos/advanced_transactions/send_micro_transaction.py';
+import TipFaucet from "../../_admonitions/_tip_faucet.md";
+
+If you need to send an amount below the minimum storage deposit, you should use a microtransaction.
+
+<TipFaucet/>
+
+## Code Example
+
+The following example will:
+
+1. Create a wallet.
+2. Get Alice's account which was [created in the first guide](../accounts_and_addresses/create_account.mdx).
+3. Create the outputs for a microtransaction.
+4. Send the micro-microtransaction using the `Account.send_micro_transaction(addresses_with_micro_amount, options)` function.
```

Comment:
Here is a list, in the others not, don't you want to align it?

Reply:
You mean the list of steps?

Reply:
Yes

Reply:
Yeah for some examples it just didn't make sense imo. But I think we should check all examples once the are all finished anyway. I'm sure @lucas-tortora can also help improve them